pg-core/src/clj/pg/core.clj

```diff
@@ -263,10 +263,12 @@
 
 (defn ->config
   "
-  Turn a Clojure map into an instance of Config.Builder.
+  Turn a Clojure map into an instance of Config via Config.Builder
   "
-  ^Config$Builder [params]
-
+  ^Config [params]
```

Comment:
I think it's OK to have both URI and other params at once, no need to throw an error. My idea was to parse the URI, if it passed, and get a map of config from it. And then, merge it with the rest of the top-level config, or vice versa.